ipatests/test_ipalib/test_util.py

```diff
@@ -0,0 +1,44 @@
+# Authors:
+#   Oleg Kozlov <okozlov@redhat.com>
+#
+# Copyright (C) 2018  Red Hat
+# see file 'COPYING' for use and warranty information
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Test the `ipalib.util` module.
+"""
+
+import os
+import pytest
```

Comment:
Minor style nit pick: In new code, we group imports by source, then sort each block alphanumerical. The first block is stdlib modules, the next block 3rd party dependencies and the last block is project code. Pytest is a 3rd party dependency.

There are few exceptions from the rule. Sometimes the import order matters because IPA has bad cyclic dependencies. Existing code should be kept as it is unless that a cleanup improves the code's readability. Non-significant changes can make backports more annoying.

## Example

```
import os
from unittest import mock

import pytest

from ipalib.util import get_pager
```